### PR TITLE
Docs: clarify that pointer events classes don't affect keyboard interaction

### PR DESCRIPTION
**Pull request by patrickhlauke**

### CODE REVIEW

**Review by mdo — COMMENTED** [review]
For this amount of content, I feel like it shouldn't be a warning. Any objections to making it standard body text here?

**Comment by patrickhlauke** [comment]
i was tending towards a warning mainly because (anecdotally) it's non-obvious to many devs who somehow just design for mouse users

File: site/content/docs/5.0/utilities/interactions.md [hunk 1]
```diff
@@ -18,10 +18,14 @@ Change the way in which the content is selected when the user interacts with it.
 
 ## Pointer events
 
-Bootstrap provides `pe-none` and `pe-auto` classes to prevent or add element interactions.
+Bootstrap provides `.pe-none` and `.pe-auto` classes to prevent or add element interactions.
 
 {{< example >}}
 <p><a href="#" class="pe-none">This link</a> can not be clicked.</p>
```
Comment (MartijnCuppens) [inline site/content/docs/5.0/utilities/interactions.md:24]: Maybe better add `tabindex="-1"` to these links?

Comment (patrickhlauke) [reply site/content/docs/5.0/utilities/interactions.md:24]: good point, oops...was so concerned with callout that i actually missed it :)

Comment (patrickhlauke) [reply site/content/docs/5.0/utilities/interactions.md:24]: fixed

**Review by MartijnCuppens — APPROVED** [review]

**Pull request merged**